src/prefect/cli/deployment.py

```diff
@@ -411,31 +410,57 @@ async def run(
         )
     parameters = {**multi_params, **cli_params}
 
-    # Parse the start time if provided
-    if start_in:
-        start_time_raw = "in " + start_in
-    elif start_at:
-        start_time_raw = "at " + start_at
-    else:
-        start_time_raw = None
-
-    if start_time_raw:
+    def get_scheduled_start_time_and_human_dt_diff(start_time_raw):
         with warnings.catch_warnings():
             # PyTZ throws a warning based on dateparser usage of the library
             # See https://github.com/scrapinghub/dateparser/issues/1089
             warnings.filterwarnings("ignore", module="dateparser")
-            start_time_parsed = dateparser.parse(
-                start_time_raw,
-                settings={
-                    "TO_TIMEZONE": "UTC",
-                    "RELATIVE_BASE": datetime.fromtimestamp(now.timestamp()),
-                },
-            )
+
+            try:
+
+                start_time_parsed = dateparser.parse(
+                    start_time_raw,
+                    settings={
+                        "TO_TIMEZONE": "UTC",
+                        "RETURN_AS_TIMEZONE_AWARE": False,
+                        "PREFER_DATES_FROM": "future",
+                        "RELATIVE_BASE": datetime.fromtimestamp(now.timestamp()),
+                    },
+                )
+
+            except Exception as exc:
+                exit_with_error(f"Failed to parse '{start_time_raw!r}': {exc!s}")
+
         if start_time_parsed is None:
             exit_with_error(f"Unable to parse scheduled start time {start_time_raw!r}.")
+
         scheduled_start_time = pendulum.instance(start_time_parsed)
+        human_dt_diff = (
+            " (" + pendulum.format_diff(scheduled_start_time.diff(now)) + ")"
+        )
+        return scheduled_start_time, human_dt_diff
+
+    if start_in and start_at:
+        exit_with_error(
+            "Expected optional `start_in` field or `start_at` field but not both."
```

Comment:
We should use the user-facing options here not our internal variable names
```suggestion
            "Only one of `--start-in` or `--start-at` can be set, not both."
```